src/persistence/Subject.ts

```diff
@@ -195,7 +195,10 @@ export class Subject {
             (this.databaseEntityLoaded === false ||
                 (this.databaseEntityLoaded && this.databaseEntity)) &&
             // ((this.entity && this.databaseEntity) || (!this.entity && !this.databaseEntity)) &&
-            this.changeMaps.length > 0
+            // ignore changes for columns which are obviously not updatable
+            this.changeMaps.some(
```

Comment:
Shouldn't this be `filter`? Some will exit as soon as true is returned, so you might ignore updatable columns as well that are after the `update: false`.
I also do not know what the conditions do tell.

Reply:
@hinogi 
Here we are checking that one or more changes for this entity exist, so early return should be fine.
I improved the comment to clarify this.